### DISCUSSION TITLE
merge #189 from emoji-selector

### DISCUSSION THREAD

**Discussion by hb0nd** [FelipeFTN/Emoji-Copy#4] [Ideas]
https://github.com/maoschanz/emoji-selector-for-gnome/pull/189

this PR is really useful, i'd love to see it merged

**Comment by FelipeFTN** [comment]
Thank you for your suggestion!
I will be merging and releasing it as soon as possible! 🚀

**Comment by FelipeFTN** [comment]
PR #189 from emoji-selector was merged https://github.com/FelipeFTN/emoji-copy/pull/6 ✨
Waiting for GNOME Extensions review and soon it will be published. :rocket:

**Comment by hb0nd** [comment]
thank you!
